docker/confd/templates/app.conf.tmpl

```diff
@@ -18,6 +18,10 @@ server {
     ssl_certificate         /etc/nginx/certs/app.crt;
     ssl_certificate_key     /etc/nginx/certs/app.key;
 
+    location =/meta.json {
+        alias /var/www/META;
```

Comment:
why not write the file as /var/www/meta.json?

Reply:
Very good point! And it's now written to `/web/meta.json` so we don't need special nginx config 💃